### PR TITLE
Caitlin ep05

### PR DESCRIPTION
**Pull request by CaitlinBate**
all code for Telemetry Reports - Array Operators, for each node.

### CODE REVIEW

**Review by jackVanish — COMMENTED** [review]
Might be good to mention we can save to other formats with the ggsave function; I don't like to leave anything implicit. Other than that, though, nothing leaps out at me.

File: _episodes/05-r-telemetry-report-array.md [hunk 1]
```diff
@@ -124,7 +127,9 @@ ggsave(plot = proj61_map, filename = "proj61_map.tiff", units="in", width=15, he
 
 ### Mapping my stations - Interactive map
 
-An interactive map can contain more information than a static map.
+An interactive map can contain more information than a static map. Here we will explore the package `plotly` to create interactive "slippery" maps. These allow you to explore your map in different ways by clicking and scrolling through the output.
```
Comment (jdpye) [inline _episodes/05-r-telemetry-report-array.md:130]: it's 'slippy' https://wiki.openstreetmap.org/wiki/Slippy_Map

**Review by jdpye — CHANGES_REQUESTED** [review]
Only very minor changes needed, great amount of flavour text in there now and this lesson stands alone very nicely.

**Comment by CaitlinBate** [comment]
changes made and pushed

File: _episodes/05-r-telemetry-report-array.md [hunk 2]
```diff
@@ -154,23 +161,26 @@ proj61_map_plotly <- proj61_map_plotly %>% add_markers(
...
 
-### How are my stations performing?
+### Summary of Animals Detected
+
+Let's find out more about the animals detected by our array! These summary statistics, created using `dplyr` functions, could be used to help determine the how successful each of your stations has been at detecting tagged animals. We will also learn how to export our results using `write_csv`.
 
-Let's find out more about the animals detected by our array!
 ~~~
-#How many detections of my tags does each station have?
+# How many of each animals did we detect from each collaborator, by species, per station
```
Comment (jackVanish) [inline _episodes/05-r-telemetry-report-array.md:180]: Should be "How many of each animal"

File: _episodes/05-r-telemetry-report-array.md [hunk 3]
```diff
@@ -124,7 +127,11 @@ ggsave(plot = proj61_map, filename = "proj61_map.tiff", units="in", width=15, he
 
 ### Mapping my stations - Interactive map
 
-An interactive map can contain more information than a static map.
+An interactive map can contain more information than a static map. Here we will explore the package `plotly` to create interactive "slippy" maps. These allow you to explore your map in different ways by clicking and scrolling through the output.
+
+First, we will set our basemap's aesthetics and bounding box and assign this information (as a list) to a geo_styling variable. Then, we choose which Deployment Metadata dataset we wish to use and identify the columns containing Latitude and Longitude, using the `plot_geo` function. Next, we use the `add_markers` function to write out what information we would like to have displayed when we hover our mouse over a station in our interactive map. In this case, we chose to use `paste` to join together the Station Name and its lat/long. Finally, we add all this information together, along with a title, using the `layout` function, and now we can explore our interactive map!
```
Comment (jackVanish) [inline _episodes/05-r-telemetry-report-array.md:132]: My preference would be to break this up and interleave each step with the appropriate code, but that's just a style thing, I leave it up to you to pull the trigger on that if you agree that it's appropriate.

**Review by jackVanish — COMMENTED** [review]
Other than a couple of things noted here, I think this looks solid.

**Comment by CaitlinBate** [comment]
made all the changes requested. going to merge

**Pull request merged**